.buildkite/auditbeat/scripts/integ-tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/env-scripts/linux-env.sh
+
+echo "--- Running Integration Tests"
+sudo chmod -R go-w auditbeat/
+
+cd auditbeat
+umask 0022
```

Comment:
why is this needed?

Reply:
It's not needed, I've removed it. I took most of this script from the unit test script, but this isn't needed here

Reply:
Buildkite-agent user has default umask=0002, meanwhile Jenkins has 0022. Without changing umask to 0022 tests are failing.